apiserver/facades/client/application/application.go

```diff
@@ -755,8 +759,11 @@ func (api *APIBase) SetCharm(args params.ApplicationSetCharm) error {
 	)
 }
 
-// SetCharmProfile a new charm's url on deployed machines for changing the profile used
-// on those machine.
+// SetCharmProfile a new charm's url on deployed machines for changing the
+// profile used on those machine.
+//
+// TODO: consider removing this public method as it is now done as part of
```

Comment:
Can we remove now? Harder to remove later after release

Reply:
yes, removing now